packages/fx-core/src/component/driver/aad/update.ts

```diff
@@ -109,7 +110,15 @@ export class UpdateAadAppDriver implements StepDriver {
         getLocalizedString(logMessageKeys.successExecuteDriver, actionName)
       );
       await progressHandler?.end(true);
-
+      const msg = getLocalizedString("core.deploy.aadManifestSuccessNotice");
```

Comment:
Since the bug is about "Teams: Update AAD manifest" command, can we add this logic to the command part (in `FxCoreImplementV3.ts`)? So when provision, users won't get this notification. I remember there's another bug that complaints provision pops up some notification. @jidddddd for your opinion.

Reply:
I find that the teams app manifest logic, which does the same thing within component.

Reply:
> Since the bug is about "Teams: Update AAD manifest" command, can we add this logic to the command part (in `FxCoreImplementV3.ts`)? So when provision, users won't get this notification. I remember there's another bug that complaints provision pops up some notification. @jidddddd for your opinion.

We could solve the pop up issue separately later, I'm still trying to figure out what to do with it because that other bug involves changing names for lifecycle stages.

Reply:
sure, thanks. 
I will send the issue bug to your first. @jidddddd